src/app/pages/StakingPage/Features/DelegationList/index.tsx

```diff
@@ -104,8 +104,11 @@ export const DelegationList = memo((props: Props) => {
       id: 'fee',
       selector: 'fee',
       sortable: true,
-      width: '100px',
-      cell: datum => (datum.validator?.fee !== undefined ? `${datum.validator.fee}%` : 'Unknown'),
+      width: '110px',
```

Comment:
:thinking: wider

Reply:
Leftover... fixed, thanks.

Reply:
I mean, I had no allegiance to 100 nor 110, but I guess all the more focused this PR is if it doesn't change. thanks